pkgs/build-support/cc-wrapper/add-hardening.sh

```diff
@@ -32,7 +32,7 @@ if [[ -n "${hardeningEnableMap[fortify3]-}" ]]; then
 fi
 
 if (( "${NIX_DEBUG:-0}" >= 1 )); then
-  declare -a allHardeningFlags=(fortify stackprotector pie pic strictoverflow format)
+  declare -a allHardeningFlags=(fortify fortify3 stackprotector pie pic strictoverflow format)
```

Comment:
is `relro` not here on purpose?

Reply:
That is a very good question. It predates me, but it's also missing `bindnow` so presumably this debug statement was meant to only print the cc-wrapper-related flags and not the bintools ones?

Reply:
BTW it would be awesome if this list of flags could be changed without causing rebuilds for `NIX_DEBUG=0` (which is the default).